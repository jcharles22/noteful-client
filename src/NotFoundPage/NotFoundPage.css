.notFoundPageContainer {
    display: flex;
    align-items: center;
    flex-direction: column;
}

.heading {
    font-weight: bold;
    font-size: 24px;
}