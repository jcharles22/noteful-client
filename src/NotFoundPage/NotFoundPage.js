import React from "react";
import "./NotFoundPage.css";

function NotFoundPage() {
  return (
    <section className="notFoundPageContainer">
      <h2 className="heading">Page Not Found</h2>
      <p>
        This page does not exist, try going back or using the navigation menu
      </p>
    </section>
  );
}
export default NotFoundPage;
