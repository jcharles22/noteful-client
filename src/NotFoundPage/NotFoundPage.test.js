import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import NotFoundPage from './NotFoundPage'

describe(`NotFoundPage component`, () => {
  it("renders without crashing", () => {
    const wrapper = shallow(<NotFoundPage />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
