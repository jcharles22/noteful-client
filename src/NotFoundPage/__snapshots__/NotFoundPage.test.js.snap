// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage component renders without crashing 1`] = `
<section
  className="notFoundPageContainer"
>
  <h2
    className="heading"
  >
    Page Not Found
  </h2>
  <p>
    This page does not exist, try going back or using the navigation menu
  </p>
</section>
`;
